apps/src/code-studio/pd/workshop_dashboard/workshop_dashboard.jsx

```diff
@@ -124,7 +124,7 @@ export default class WorkshopDashboard extends React.Component {
               component={DailySurveyResultsLoader}
             />
             <Route
-              path="daily_survey_results_foorm(/:workshopId)"
```

Comment:
Was this route name unused before this PR?

Reply:
it was what we used for testing, I wanted to remove 'foorm' from the end user experience so I made the url more generic